tests/pytests/test_info_modules.py

```diff
@@ -17,6 +17,28 @@ def info_modules_to_dict(conn):
         info[section_name][data[0]] = data[1]
   return info
 
+def get_search_field_info(type: str, count: int, sortable_count: int = 0, no_index_count: int = 0, index_errors: int = 0):
+    return f'{type}={count}' \
+           f'{",Sortable=" + str(sortable_count) if sortable_count else ""}' \
+           f'{",NoIndex=" + str(no_index_count) if no_index_count else ""}' \
+           f',IndexErrors={index_errors}'
+
+def get_search_tag_field_info(count:int,
+                              sortable_count: int = 0,
+                              no_index_count:int = 0,
+                              case_sensitive_count:int = 0,
+                              index_errors:int = 0):
+  return f'Tag={count}' \
+         f'{",Sortable=" + str(sortable_count) if sortable_count else ""}' \
+         f'{",NoIndex=" + str(no_index_count) if no_index_count else ""}' \
+         f'{",CaseSensitive=" + str(case_sensitive_count) if case_sensitive_count else ""}' \
```

Comment:
I know it is not a part of this PR but do we have a tests that checks what happens when the values are 0?

Reply:
yes 
fields options like sortable etc, are only printed when set (you can see here that the string is empty)
`IndexErrors` is always printed and is 0 zero in some of the tests